tests/pytests/test_dipoles.py

```diff
@@ -0,0 +1,41 @@
+import pytest
```

Comment:
need to add this file to https://github.com/psi4/psi4/blob/master/psi4/CMakeLists.txt#L257 to get it installed.

Reply:
Reminder that we need the procedure for adding pytests documented. I'm surprised that's the only thing that needed changing.